src/AspNetStandard.Diagnostics.HealthChecksWcf/WcfHeathCheckRouteExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ServiceModel.Activation;
+using System.Web.Routing;
+
+namespace AspNetStandard.Diagnostics.HealthChecksWcf
+{
+    public static class WcfHeathCheckRouteExtension
+    {
+        public static WcfHealthCheckBuilder AddWcfHealthCheck()
```

Comment:
Deixaria explícito desta maneira:

```suggestion
        public static WcfHealthCheckBuilder AddWcfHealthCheckEndpoint()
```

Reply:
-> Acho que não tem problema ser Create, mas no padrão do pra web-api está com Add

Reply:
Se o padrão é Add então é melhor manter. Acho estranho pq na prática estamos criando.